analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -394,25 +371,12 @@ def behavior(self):
         return behavior
 
 
-# %% [markdown]
-# ### "Fileset" construction and metadata
-#
-# Here, we gather all the required information about the files we want to process: paths to the files and asociated metadata.
-
-# %% tags=[]
-fileset = utils.construct_fileset(N_FILES_MAX_PER_SAMPLE, use_xcache=False, af_name=AF_NAME)  # local files on /data for ssl-dev
-
-print(f"processes in fileset: {list(fileset.keys())}")
-print(f"\nexample of information in fileset:\n{{\n  'files': [{fileset['ttbar__nominal']['files'][0]}, ...],")
-print(f"  'metadata': {fileset['ttbar__nominal']['metadata']}\n}}")
-
-
```

Comment:
Is there a reason the order flipped with the query and the fileset construction?

Reply:
I flipped it because I put the servicex step right after fileset construction. But I think it makes sense to keep the servicex stuff together after fileset construction so I will change it.

Reply:
Fixed in [9587d6c](https://github.com/iris-hep/analysis-grand-challenge/pull/103/commits/9587d6c0f7766965c0f95282c5a78fbcf1c14001)